internal/utils/names.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package utils
+
+import (
+	"runtime"
+	"strings"
+)
+
+func GetPackageAndName(pc uintptr) (suite string, name string) {
```

Comment:
I would add a doc here with an output example

Reply:
done in 9bfaa52